daemon/info.go

```diff
@@ -75,7 +75,7 @@ func (daemon *Daemon) SystemInfo() (*types.Info, error) {
 		DriverStatus:       daemon.layerStore.DriverStatus(),
 		Plugins:            daemon.showPluginsInfo(),
 		IPv4Forwarding:     !sysInfo.IPv4ForwardingDisabled,
-		BridgeNfIptables:   !sysInfo.BridgeNfCallIptablesDisabled,
+		BridgeNfIptables:   !sysInfo.BridgeNfCallIPtablesDisabled,
```

Comment:
As long as the serialization is not changed, it won't change over-the-wire compatibility. But docker/docker and other engine-api users would need to be kept in sync to compile with the new engine-api changes. I'll defer to others on whether this is worth the trouble. We've had some trouble with engine-api getting ahead of docker/docker recently, which really compilcate PRs that involve engine-api changes.